data/pages-content/cfp.md

```diff
@@ -0,0 +1,164 @@
+---
+title: CFP
+subtitle: EuroPython Call for Proposals
+---
+# Call for Proposals
+
+<div style={{textAlign: "center",  marginBottom: 10}}>
+**EuroPython 2024 Call for Proposals (CFP) is Open.**
+
+It will be running between
+**Wednesday, 14th February 2024** and **Wednesday, 6th March 2024**
+[Anywhere on Earth](https://en.wikipedia.org/wiki/Anywhere_on_Earth)
+</div>
+
+---
+
+EuroPython reflects the colourful and diverse backgrounds, cultures and
+interests of our community, so you (yes, you!) should go for it: **propose
+something and represent!**
+
+No matter your level of Python or public speaking experience, EuroPython's job
+is to help you bring yourself to our community so we all flourish and benefit
+from each other's experience and contribution.
+
+If you’re thinking, “but they don’t mean me”, then we especially mean **YOU**.
+
+* If you’re from a background that isn't usually well-represented in most
+  Python groups, get involved - we want to help you make a difference.
+
+* If you’re from a background which is well-represented in most Python groups,
+  get involved - we want your help making a difference.
+
+* If you’re worried about not being technical enough, get involved - your fresh
+  perspective will be invaluable.
+
+* If you think you’re an imposter, join us - many of the EuroPython organisers
+  feel this way.
+
+* This is a volunteer led community, so join us - you will be welcomed with
+  friendship, support and compassion.
+
+You are welcome to share your questions and ideas with our programme team at
+[programme@europython.eu](mailto:programme@europython.eu).
+
+## The Conference Format ##
+
+The conference will feature around 16 Tutorials/Workshops and 120 talk sessions:
+
+* **Tutorials/Workshops (July 8th and 9th)**: hands-on **180-mins sessions** with
+  40-100 people
+
+* **Talks (July 10th, until 12th)** : **30 or 45 minutes** : presentations of specific
+  and general interest to the European Python community (including a dedicated
+  PyData track)
```

Comment:
Is this intended? I'm not a native speaker but "45 minute" certainly reads strange to me. It might be my ignorance, but besides '1 minute' I was under the impression all the other amounts included an 's'.

Reply:
This was intended but _I_ may be wrong.
When counting minutes by themselves, you do say 1 minute, 2 minuteS, 3 minuteS, etc, but in this context I thought you dropped the s.
I can't explain why. Maybe I'm wrong!

Reply:
When 'minute' is used as a compound adjective, such as in 'a 45-minute presentation,' it is singular and should have a hyphen. However, when used as a noun, it should be plural when referring to more than one minute, as in 'the presentation will last 45 minutes.' Because '30-or-45-minute talks' may appear awkward, changing it to a noun format like 'talks: 30 or 45 minutes' might improve readability. In this case, 'minute' is not modifying 'talks,' so it should be plural. This also applies to the abbreviated format 'min.'

Reply:
Thank you so much for explaining @hypha!